app/lib/publisher/backend.dart

```diff
@@ -169,6 +206,41 @@ class PublisherBackend {
     });
   }
 
+  /// A callback from consent backend, when a consent is granted.
+  Future inviteConsentGranted(String publisherId, String userId) async {
+    await _db.withTransaction((tx) async {
+      final key = _db.emptyKey
+          .append(Publisher, id: publisherId)
+          .append(PublisherMember, id: userId);
+      final list = await tx.lookup<PublisherMember>([key]);
+      final member = list.single;
+      if (member == null) {
+        throw NotFoundException('Membership invite was deleted.');
```

Comment:
Create a custom constructor please :)

Reply:
This check will be removed once we remove isPending.